src/main/java/uk/org/tombolo/execution/DataExportEngine.java

```diff
@@ -101,4 +103,38 @@ private void importDatasource(ImporterMatcher forceImports, DatasourceRecipe dat
 				forceImports.doesMatch(datasourceSpec.getImporterClass())
 		);
 	}
+
+	private Importer initialiseImporter(String importerClass, String configFile) throws Exception {
+		Config importerConfiguration = null;
+		if (configFile != null && !"".equals(configFile)) {
+			importerConfiguration = ConfigUtils.loadConfig(
+					AbstractRunner.loadProperties("Configuration file", configFile));
+
+		}
+		return (Importer) Class.forName(importerClass).getDeclaredConstructor(Config.class).newInstance(importerConfiguration);
+	}
+
+	private String verifyProvider() throws Exception {
+		String validProvider = null;
+
+		File specificationFile = new File(DataExportRunner.executionSpecPath);
+		if (!specificationFile.exists()) return validProvider;
```

Comment:
This check is already done in validateSpecification() that is called before the execute.

Reply:
This is necessary to see if the method is invoked by tests or actual recipe file. As tests wont have a file but would execute this method